contracts/PriceOracle/PriceOracle.sol

```diff
@@ -0,0 +1,214 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.7;
+
+import { Ownable } from "../Ownable.sol";
```

Comment:
Consider using 2-step ownable if it's acceptable by Compound

Reply:
I believe the internal [Ownable contract](https://github.com/smartcontractkit/open-oracle/blob/master/contracts/Ownable.sol) we use is also 2-step but I see the benefits of using the official version. The interfaces line up so don't think there is an issue with using Ownable2Step. I've updated it in the latest commit